docs/source/sctool/partials/sctool_backup_list.yaml

```diff
@@ -45,11 +45,11 @@ options:
     Specifies maximal snapshot `date` expressed in RFC3339 form or 'now[+duration]', ex. 'now+3d2h10m'.
     Valid units are:
 
-    'd' - days
-    'h' - hours
-    'm' - minutes
-    's' - seconds
-    'now' - happens immediately
+    * ``d`` - days
```

Comment:
@mmatczuk Could you please  update the code in `master` to generate this output (applies to all lists)? Sphinx needs the character ``*`` to format it as a list.